docs/pages/gettingStarted.md

```diff
@@ -30,17 +49,18 @@ nav_order: 2
    @using dymaptic.GeoBlazor.Core.Components.Views
    @using dymaptic.GeoBlazor.Core.Objects
    ```
-
 6. Create a new Razor Component in the `Pages` folder, or just use `Index.razor`. Add a `MapView`. Give it basic
    parameters to ensure that it can render.
-    ```html
-    @page "/"
-    
-    <MapView Longitude="_longitude" Latitude="_latitude" Zoom="11" Style="height: 400px; width: 100%;"> 
-    </MapView>
-    ```
-7. Within the `MapView`, define a `Map`. To load a pre-generated map from ArcGIS Online or a Portal, get the Portal Id 
+
+   ```html
+   @page "/"
+
+   <MapView Longitude="_longitude" Latitude="_latitude" Zoom="11" Style="height: 400px; width: 100%;"> 
+   </MapView>
+   ```
+7. Within the `MapView`, define a `Map`. To load a pre-generated map from ArcGIS Online or a Portal, get the Portal Id
```

Comment:
Couple of things here. First, I would say "map from ArcGIS Online or Portal". Saying "a Portal", while correct, sounds weird.

Second, Portal ID and Map ID are different things. If you want the ID of the organization, then that is the Portal ID. That would be the ID for ArcGIS Online or for Portal. If you want the ID of the map, then that is an Item ID. Regardless if the item is in ArcGIS Online or Portal. Hope this makes sense.